server/configs/webapps/embedded/labkey_server.service

```diff
@@ -21,7 +21,7 @@ ExecStart=<JAVA_HOME>/bin/java \
     -Xms2G \
     -Xmx2G \
     -XX:+HeapDumpOnOutOfMemoryError \
-    -XX:ErrorFile=/labkey/labkey/logs/error_%p.log \
+    -XX:ErrorFile=$LABKEY_HOME/logs/error_%p.log \
```

Comment:
Any reason not to just remove the `ErrorFile` parameter entirely? You mentioned that it is unlikely to every be used and even more unlikely to be useful if it is.
Also, we don't include it in the example Windows service. I would think that the Windows and Linux examples should be functionally identical in terms of startup parameters.

Reply:
That's fine with me. In the absence of this parameter, the JVM will write to hs_err_<pid>.log in the working directory.